client/my-sites/stats/promo-cards/index.jsx

```diff
@@ -2,49 +2,70 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { MobilePromoCard } from '@automattic/components';
 import { translate } from 'i18n-calypso';
 import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import blazeDropDownIllustration from 'calypso/assets/images/illustrations/blaze-drop-down.svg';
 import wordpressSeoIllustration from 'calypso/assets/images/illustrations/wordpress-seo-premium.svg';
 import PromoCardBlock from 'calypso/blocks/promo-card-block';
 import DotPager from 'calypso/components/dot-pager';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 import './style.scss';
 
 const EVENT_YOAST_PROMO_VIEW = 'calypso_stats_wordpress_seo_premium_banner_view';
+const EVENT_BLAZE_PROMO_VIEW = 'calypso_stats_blaze_banner_view';
 const EVENT_MOBILE_PROMO_VIEW = 'calypso_stats_traffic_mobile_cta_jetpack_view';
 
-export default function PromoCards( { isJetpack, isOdysseyStats, slug } ) {
+export default function PromoCards( { isOdysseyStats, slug } ) {
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const jetpackNonAtomic = useSelector(
+		( state ) => isJetpackSite( state, selectedSiteId ) && ! isAtomicSite( state, selectedSiteId )
+	);
+
 	// Yoast promo is disabled for Odyssey & self-hosted.
 	// Mobile apps promos are always shown.
-	const showYoastPromo = ! isOdysseyStats && ! isJetpack;
-	// Handle initial view event if not using the DotPager UI.
-	// We don't worry about the Yoast card as it sends on mount.
-	useEffect( () => {
```

Comment:
Replaced this logic [with a simpler one](https://github.com/Automattic/wp-calypso/pull/71286/files#diff-d0adee9cff76fc082d6cdc6ebfa2dde684cfa28d52007d8c69599eb26b943f87R47).